src/wazuh.py

```diff
@@ -220,14 +223,17 @@ def _get_current_configuration_url_branch(container: ops.Container) -> str:
 
     Returns:
         The repository branch.
+
+    Raises:
+        WazuhInstallationError: if an error occurs while fetching repository configuration.
     """
     process = container.exec(["git", "-C", REPOSITORY_PATH, "rev-parse", "--abbrev-ref", "HEAD"])
     branch = ""
     try:
         branch, _ = process.wait_output()
+        return branch.rstrip()
     except ops.pebble.ExecError as ex:
-        logging.debug(ex)
-    return branch.rstrip()
```

Comment:
It looks like we're changing the behavior of the charm to fall into Error state - would this be intended for the charm to retry afterwards? (since git issue is likely due to proxy/network config issues)?